react/game/src/app/components/Router/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { Switch, Route } from 'react-router-dom';
+import { ConnectedRouter } from 'connected-react-router';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import LIST from '../../../constants/routes';
+import { history } from '../../../redux/store';
+import Login from '../../screens/Login';
+import Game from '../../screens/Game';
+import About from '../../screens/About';
+import WithLoading from '../WithLoading';
+import PrivateRoute from '../PrivateRoute';
+
+class Router extends Component {
```

Comment:
Since this component its became a dumb component, it's no longer necessary that it be a class. Convert this component to a functional component